scripts/golos-boot-sequence/golos-boot-sequence.py

```diff
@@ -235,6 +235,7 @@ def createGolosAccounts():
     updateAuth('cyber',       'createuser', 'active', ['GLS5a2eDuRETEg7uy8eHbiCqGZM3wnh2pLjiXrFduLWBKVZKCkB62'], [])
     linkAuth('cyber', 'cyber', 'newaccount', 'createuser')
     linkAuth('cyber', 'gls.vesting', 'open', 'createuser')
+    linkAuth('cyber', 'gls.referral', 'addreferral', 'createuser')
```

Comment:
`cyber` is the system user of the blockchain. He has no relation to Golos dApp smart-contracts.
For correct work, it is necessary for the `gls.referral` user to create a special authority, the key from which will be loaded into the site. It's also need to allow the `gls.referral` user to perform a method addreferral` action with this authority.